hermes/views.py

```diff
@@ -181,9 +180,137 @@ def get(self, request, *args, **kwargs):
          topic: <kafka topic to post message to>, 
          submitter: <submitter of the message>,
-         authors: <Text full list of authors on a message>
+         authors: <Text full list of authors on a message>,
          message_text: <Text of the message to send>,
-         data: {<Unparsed json data dict>}
+         data: {
+            references: [
+                {
+                    source:
+                    citation:
+                    url:
+                },
+                ...
+            ],
+            extra_data: {
+                key1: value1,
+                key2: value2,
+                ...
+            },
+            event_id: <Nonlocalized event_id this message relates to>,
+            targets: [
+                {
+                    name: <Target name>,
+                    ra: <Target ra in decimal or sexigesimal format>,
+                    dec: <Target dec in decimal or sexigesimal format>,
+                    ra_error: <Error of ra>,
+                    dec_error: <Error of dec>,
+                    ra_error_units: <Units for ra error>,
+                    dec_error_units: <Units for dec error>,
+                    pm_ra: <RA proper motion in arcsec/year>,
+                    pm_dec: <Dec proper motion in arcsec/year>,
+                    epoch: <Epoch in MJD>,
+                    orbital_elements: {
+                        epoch_of_elements: <>,
+                        orbinc: <>,
+                        longascnode: <>,
+                        argofperih: <>,
+                        eccentricity: <>,
+                        meandist: <>,
+                        meananom: <>,
+                        perihdist: <>,
+                        epochofperih: <>
+                    },
+                    discovery_info: {
+                        reporting_group: <>,
+                        discovery_source: <>,
+                        transient_type: <Type of source, one of PSN, nuc, PNV, AGN, or Other>,
+                        proprietary_period: <Duration that this discovery should be kept private>,
+                        proprietary_period_units: <Units for proprietary period, Days, seconds, Years>
+                    },
+                    redshift: <>,
+                    host_name: <Host galaxy name>,
+                    host_redshift: <>,
+                    aliases: [
+                        'alias1',
+                        'alias2',
+                        ...
+                    ],
+                    group_associations: <String of group associations for this target>
+                }
+            ],
+            photometry: [
+                {
+                    target_index: <Index of target list that this photometry relates to. Can be left out if there is only one target>,
+                    date_obs: <Date of the observation, in a parseable format or JD>,
+                    telescope: <Discovery telescope>,
```

Comment:
Should probably replace "Discovery" with "Observation" for Telescope and Instrument.